### PR TITLE
RADseq denovo tuto

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

**Comment by yvanlebras** [comment]
Denovo radseq tuto second pass finished! Can you look at it and accept PR if possible ?

File: topics/sequence-analysis/tutorials/de-novo-rad-seq/tutorial.md [hunk 1]
```diff
@@ -177,35 +185,70 @@ Run `Stacks: De novo map` Galaxy tool. This program will run ustacks, cstacks, a
...
-> **Stacks: populations** {% icon tool %}: Run the last step of **Stacks: De novo map** pipeline specifying data filtering options (minimum percentage of individuals in a population required to process a locus for that population: 0.75 , output options (VCF and Structure) and enabling SNP and haplotype-based F statistics calculation.
+> 1. **Stacks: populations** {% icon tool %} with the following parameters:
+>    - Section **Data filtering options**
+>      - *"Minimum percentage of individuals in a population required to process a locus for that population"*: `0.75`
+>    - Section **Output options** (VCF and Structure) and enabling SNP and haplotype-based F statistics calculation.
+>      - *"Output results in Variant Call Format (VCF)"*: `Yes`
+>      - *"Output results in Structure Format"*: `Yes`
+>    - *"Enable SNP and haplotype-based F statistics"*: `Yes`
 >
 >    ![The output of the populations tool](../../images/RAD4_Population_Genomics/denovo/populations_log.png)
```
Comment (shiltemann) [inline topics/sequence-analysis/tutorials/de-novo-rad-seq/tutorial.md]: is this the correct image? the alt text implies it should be the output of the tool? but it is a screenshot of the job information menu?

**Comment by shiltemann** [comment]
thanks for the updates @yvanlebras! I reformatted some of the hands-on boxes, please check if everything looks ok to you :)

File: topics/sequence-analysis/tutorials/de-novo-rad-seq/tutorial.md [hunk 2]
```diff
@@ -164,8 +168,12 @@ For quality control, we use similar tools as described in [NGS-QC tutorial]({{si
 >    > > The read length is 32 bp
 >    > {: .solution }
 >    {: .question}
+>
+> 2. **MultiQC** {% icon tool %}: Run MultiQC on FastQC results to better see quality information over samples.
 {: .hands_on}
 
+![MultiQC output](../../images/RAD4_Population_Genomics/multiqc.PNG)
```
[the remark this answers is not in the document]
Comment (shiltemann) [reply topics/sequence-analysis/tutorials/de-novo-rad-seq/tutorial.md]: ah, it is in the other PR, that's ok then :)

**Comment by shiltemann** [comment]
thanks @yvanlebras!

**Pull request merged**